libraries/chain/finality/qc.cpp

```diff
@@ -1,5 +1,6 @@
 #include <eosio/chain/finality/qc.hpp>
 #include <eosio/chain/finality/vote_message.hpp>
+#include <eosio/chain/block_header_state.hpp>
```

Comment:
Why this include?

Reply:
compiler cannot find `block_header::num_from_id()`